NEWS.md

```diff
@@ -1,3 +1,13 @@
+## 2.0.0 - Released
```

Comment:
Do we need the `- Released` part? Is it a new standard?

Reply:
not a new standard - I was borrowing the format from another module, which I think is more complete.  I admin it does look silly w/o the context, so I compromised and used the release date like was done in previous versions.  The next version was added with the "unreleased" indicator.